### ISSUE TITLE
Method to encrypt value using public key or private key (more flexible createSignature)

### ISSUE THREAD

**Issue by hansyulian** [SelfLender/react-native-biometrics#41]
I know this is the package that is aimed to be used for sending the public key to the server. But this time, i just want to use this to encrypt and store many other things.

So i wish we can have the 
```
publicKeyEncryptMessage(payload,publicKey)
privateKeyEncryptMessage(payload,promptMessage)
publicKeyDecryptMessage(encryptedPayload,publicKey)
privateKeyDecryptMessage(encryptedPayload,promptMessage)
```

since the algorithm to encrypt and decrypt is the same for RSA, then we can just simplify to
```
processPublicKey(payload,publicKey) 
processPrivateKey(payload, promptMessage) 
```
and all of them return the value after RSA process.

So we can have a library that use the biometrics for anything we want to make it secure and not only limited to the scenario that the public key must be stored in server.

**Comment by mTRx0** [comment]
We've implemented this feature in our internal fork and would like to merge it. Would this still be a good addition to the package?

**Comment by jayfunk** [comment]
Definitely! This feature has been asked for a few times and I think it would be a nice addition. On Nov 8, 2022, at 06:51, Fabian ***@***.***> wrote:﻿
We've implemented this feature in our internal fork and would like to merge it. Would this still be a good addition to the package?
